apps/web/src/pages/api/forms/organisation.ts

```diff
@@ -91,6 +91,12 @@ export default withApiHandler<ExtendedNextApiRequest>(Roles.ContactManager, asyn
               data: {
                 isDeleted: false,
                 updatedBy: { connect: { id: contactManagerUserId } },
+                user: {
+                  update: {
+                    registrationToken,
+                    registrationConfirmed: false,
```

Comment:
For an existing user in SE who already has an IDG account, I'm pretty sure this is going to set these values and send the registration email to users who already have an IDG account.

Reply:
@Jaffacakes82 I wasn't aware of a non-registration email?  In the email templates there is only add user, remove user and reminder.  Existing accounts are sent the email but without appending the registration token

Appending the token and following the email gives a "Already registered with IDG" message